packages/kcms/src/match/adaptor/validator/match.ts

```diff
@@ -23,6 +23,9 @@ const RunResultSchema = z.object({
 const PreSchema = z.object({
   id: z.string().openapi({ example: '320984' }),
   matchCode: z.string().openapi({ example: '1-3' }),
+  departmentType: z
+    .enum(pick(config.departments, 'type'))
```

Comment:
#415 で`departmentTypes`が生えたので、そちらを使えます。
```suggestion
    .enum(config.departmentTypes)
```
L38も同様です。
(他の箇所も順々に直す？）